common/option-model.cpp

```diff
@@ -510,6 +510,7 @@ bool option_model::draw_checkbox( notifications_model & model,
                                                << ( bool_value ? "ON" : "OFF" ) << ")" );
 
         set_option( opt, bool_value ? 1.f : 0.f, error_message );
+        *invalidate_flag = true;
```

Comment:
Please protect nul reference by adding:
`if ( invalidate_flag ) *invalidate_flag = true;`

I guess I dropped it somehow :(
I an curios, where was that used?

Reply:
Oh I found that, tricky..
Good catch!